deployment/sw360baseimage/setupProxy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# Copyright Bosch Software Innovations GmbH, 2016.
+# Part of the SW360 Portal Project.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+# this script does
+#   - unset the proxies, if they were set to "None"
+#   - set the proxies to "${gateway}:3128", if they were set to CNTLM_ON_GATEWAY
+
+if [ "$http_proxy" ]; then
+    if [ "$http_proxy" = "None" ]; then
```

Comment:
Or maybe it is docker-compose at line https://github.com/sw360/sw360chores/blob/1e6ef10ce821d4e92332c2dd58b97e42d3f167b7/deployment/docker-compose.yml#L32